test/Reverts.ts

```diff
@@ -23,13 +28,22 @@ describe("Reverts test", function () {
     jommys = await ethers.getImpersonatedSigner(
       "0x0AeaC6D1424EA6d0F87123A50CA5eEc9f16108c5",
     );
-    MRC = await ethers.getContractAt(
-      "IMRC",
-      "0xeF453154766505FEB9dBF0a58E6990fd6eB66969",
-    );
+    if (env.TEST_LOCAL_BLOCKCHAIN) {
+      const { e7lContract, mrcContract } = await deployContracts();
+      E7L = e7lContract;
+      MRC = mrcContract;
+
+      await MRC.mint(1);
+      await MRC.connect(yonathan).mint(1);
+    } else {
+      MRC = await ethers.getContractAt(
+        "MRCRYPTO",
+        "0xeF453154766505FEB9dBF0a58E6990fd6eB66969",
+      );
```

Comment:
It's better to create a new test that cover this functionality. Don't reuse tests.

Imagine that in the future the possibilities of testing in different networks/local/test net etc... are further parameterized, this would mean that the maintainability and legibility of the tests would not be correct.

Reply:
It will be great, thanks for the feedback